dotcom-rendering/src/web/components/Callout/CalloutShare.tsx

```diff
@@ -0,0 +1,69 @@
+import { css } from '@emotion/react';
+import { textSans } from '@guardian/source-foundations';
+import { Button, SvgShareCallout } from '@guardian/source-react-components';
+import { useState } from 'react';
+import { decidePalette } from '../../lib/decidePalette';
+
+const shareCalloutStyles = css`
+	display: flex;
+	align-items: center;
+`;
+
+const shareCalloutTextStyles = css`
+	display: inline-block;
+	${textSans.xsmall()}
+`;
+
+const shareCalloutLinkStyles = (format: ArticleFormat) =>
+	css`
+		color: ${decidePalette(format).text.richLink};
+		border-bottom: 1px solid ${decidePalette(format).text.richLink};
+		text-decoration: none;
+		font-weight: normal;
+	`;
+
+interface Props {
+	format: ArticleFormat;
+}
+
+export const CalloutShare = ({ format }: Props) => {
+	const [isCopied, setIsCopied] = useState(false);
+
+	const onShare = async () => {
+		const url = window.location.href;
+		if ('clipboard' in navigator) {
+			await navigator.clipboard.writeText(url);
+			setIsCopied(true);
+			setTimeout(() => setIsCopied(false), 2000);
+		}
+	};
+	if (typeof window === 'undefined' || typeof navigator === 'undefined')
+		return <></>;
+
+	return (
+		<>
+			<div css={shareCalloutStyles}>
+				<div
+					css={css`
+						width: 45px;
+					`}
+				>
+					<SvgShareCallout />
+				</div>
+				<div css={shareCalloutTextStyles}>
+					Know others who are affected?{' '}
+					<Button
+						size="xsmall"
+						priority="subdued"
+						onClick={onShare}
+						css={shareCalloutLinkStyles(format)}
+					>
+						{' '}
+						Please share this callout.
+					</Button>
+					{isCopied && <em> Link copied to clipboard</em>}
```

Comment:
Wrapping the link an `em` tag means the screen reader will read this to anyone using a screen reader. Thanks for the tip there @sophie-macmillan

Reply:
Not quite! It needs to have the [role="alert"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role) attribute. See the [same component in AR](https://github.com/guardian/dotcom-rendering/pull/6800/files#diff-7e7918a3b84020e0100f847598aa1ae7273ec41877d4156b63293921bc35a96eR41) for reference (also worth noting that the styling has been updated so its not italic and is the same as the supporting text on the fields)

Reply:
Cool thanks - will implement here too